wiki/People/The_Team/Beatmap_Nominators/ko.md

```diff
@@ -0,0 +1,254 @@
+---
+tags:
+  - BN
+  - BNG
+---
+
+# 비트맵 노미네이터
+
+*이 문서들도 확인해보세요: [비트맵 노미네이터 규칙](Rules), [비트맵 노미네이터 행동강령](Expectations), [비트맵 노미네이터를 위한 일반적인 정보](General_Information), 그리고 [비트맵 노미네이터가 되는 방법](Becoming_a_Beatmap_Nominator)*
+
+**비트맵 노미네이터** (***BN***)는 [비트맵 지명](/wiki/Beatmap_ranking_procedure)을 담당하는 그룹입니다. 이들은 포럼과 비트맵 토론 창에서 보라색 이름으로 보여집니다.
+
+## 역할과 책임
+
+BN은 주로 [보류 중 상태인 비트맵](/wiki/Beatmap/Category#work-in-progress-and-pending)을 [모딩](/wiki/Modding)하고 승인해 랭킹 사이클로 올리는 일을 담당합니다. 이들은 매핑이나 모딩에 관련된 내용들을 알려주거나 요청을 받습니다. 이미 지명받은 비트맵에 랭킹 크리테리아를 위반한 내용이 없으나 반드시 수정되어야 하는 문제가 있다고 생각하면 누구나 [비트맵 Veto](Beatmap_Veto)를 걸 수 있습니다.
```

Comment:
could be "Ranking criteria" or "랭크 기준" instead of "랭킹 크리테리아"
Currently It shows Ranking criteria on wiki
https://github.com/ppy/osu-wiki/blob/master/wiki/Ranking_Criteria/ko.md